src/reducer/labelSlice.js

```diff
@@ -0,0 +1,48 @@
+import { createSlice, createAsyncThunk, createEntityAdapter } from '@reduxjs/toolkit';
+import APIHelper from 'helpers/APIHelper';
+
+export const actions = {
+  fetchAllLabels: createAsyncThunk('labels/fetchAllLabels', async (params, { rejectWithValue }) => {
```

Comment:
Let us follow the rails naming convention to make it easier.

index - instead of Fetch all
show - fetchOne

and son on.